### PR TITLE
Int b 19300

### PR DESCRIPTION
**Pull request by brooklyn-welsh**
## Summary

This branch contains 3 big changes:

1. The "Haul Price" item in the "Incentives" section of the final PPM Closeout Review screen will now update to read either "Linehaul Price" or "Shorthaul Price" depending on whether or not the move happens within the same Zip3.
2. This branch also fixed a bug in the pricing where items were always using the *linehaul* pricer for things like gross incentive, domestic area price, etc., even if a shipment would be classified as a shorthaul.
3. "Full Pack/Unpack Charge" has now been separated in to separate "Packing" and "Unpacking" charges.

Other minor changes:
1. "(Short/Line)haul Fuel Surcharge" has been changed to "(Short/Line)haul Fuel Rate Adjustment"

### How to test

1. Create a move that goes to any two zip codes with the same first 3 digits. Where you would normally check "use my new duty location", you can instead put in an address with the same Zip3 (e.g. 90210 -> 90245).
4. Go through the normal process to get to PPM closeout (sign, approve as SC, upload PPM documents).
5. As an SC, access the move through the closeout queue, go to the last "Confirm" screen and verify that you see "shorthaul price", "Shorthaul Fuel Rate Adjustment", and the separate "Packing Charge" and "Unpacking Charge" fields  as shown here:
Screenshot 2024-05-23 at 2 15 09 PM

6. If you want to test that linehaul price will show up, find the `actual_pickup_postal_code` and `actual_destination_postal_code` columns for your move in the `ppm_shipments` table. Edit the destination code to "30183".
Screenshot 2024-05-22 at 5 02 47 PM

7. Refresh and continue back to the "Confirm" screen, you should now see that "Linehaul Price" and "Linehaul Fuel Rate Adjustment" shows up instead, and you should notice a substantial difference in price.

Screenshot 2024-05-22 at 5 04 21 PM

### CODE REVIEW

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

File: src/components/Office/PPM/PPMHeaderSummary/HeaderSection.jsx [hunk 1]
```diff
@@ -33,6 +33,17 @@ const getSectionMarkup = (sectionInfo) => {
...
+  const renderHaulTypeSwitch = (haulType) => {
+    switch (haulType) {
+      case 'Linehaul':
+        return 'Linehaul Price';
+      case 'Shorthaul':
+        return 'Shorthaul Price';
+      default:
+        return <Alert>Received unknown value for haul type!</Alert>;
+    }
+  };
```
[the remark this answers is not in the document]
Comment (brooklyn-welsh) [reply src/components/Office/PPM/PPMHeaderSummary/HeaderSection.jsx]: Yep, sorry meant to open this as a draft and not mark as ready at the time. This particular section of code was refactored, but I've written a test file that tests the rest of the code at `HeaderSection.test.jsx`

**Review by WeatherfordAaron — APPROVED** [review]
lgtm 👽

File: .github/workflows/analyze-bundle.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@ jobs:
 
       - name: Check out branch
         if: steps.cache-bundle-stats.outputs.cache-hit != 'true'
-        uses: actions/checkout@v4.1.2
+        uses: actions/checkout@v4.1.6
```
Comment (danieljordan-caci) [inline .github/workflows/analyze-bundle.yml:61]: Are these changes cool to go to `integrationTesting` @deandreJones?

Comment (deandreJones) [reply .github/workflows/analyze-bundle.yml:61]: yes- generally - dependabot library updates go straight to main
so someone is bound to get it in there PR if they do it right (pulling in main)

**Review by danieljordan-caci — APPROVED** [review]

**Pull request merged**